apps/src/sites/studio/pages/curriculum_catalog/index.js

```diff
@@ -36,6 +37,7 @@ $(document).ready(function () {
         languageNativeName={languageNativeName}
         isSignedOut={isSignedOut}
         isTeacher={isTeacher}
+        selfPacedPlCourseOfferings={selfPacedPlCourseOfferings}
```

Comment:
Since we added the field editor for a course offering's self-paced PL, couldn't that be part of `curriculaData` and not require another paramter?